libpod/container_internal_linux.go

```diff
@@ -2225,6 +2249,92 @@ func (c *Container) copyOwnerAndPerms(source, dest string) error {
 	return nil
 }
 
+// harmlessHostPath validates if a host path is harmless and can be modified
+func (c *Container) harmlessHostPath(path string) bool {
```

Comment:
Nit, change the function to dangerousHostPath() and return true.

Reply:
This has been changed in containers/common